PublicFolders/ValidateEXOPFDumpster.ps1

```diff
@@ -418,6 +418,7 @@ function QuitEXOSession {
             LogError -CurrentStatus $CurrentStatus -Function "Disconnecting from EXO" -CurrentDescription $CurrentDescription
             Write-Host "`nOutput was exported in the following location: $ExportPath" -ForegroundColor Yellow
             Start-Sleep -Seconds 3
+            Exit
```

Comment:
you need to make it a lower case `exit` to follow our code formatting polices.

Reply:
Done.